scene/main/window.cpp

```diff
@@ -630,7 +630,6 @@ void Window::_event_callback(DisplayServer::WindowEvent p_event) {
 		case DisplayServer::WINDOW_EVENT_DPI_CHANGE: {
 			_update_viewport_size();
 			_propagate_window_notification(this, NOTIFICATION_WM_DPI_CHANGE);
-			emit_signal(SNAME("dpi_changed"));
```

Comment:
Probably we should have `ADD_SIGNAL(MethodInfo("dpi_changed"));` added to the `bind_methods` instead of removing it, since all other window events have both signal and notification.